lms/djangoapps/django_comment_client/base/views.py

```diff
@@ -94,6 +95,11 @@ def track_created_event(request, event_name, course, obj, data):
     track_forum_event(request, event_name, course, obj, data)
 
 
+def _add_truncated_title(event_data, full_title):
+    event_data['title_truncated'] = len(full_title) > TRACKING_MAX_FORUM_TITLE
```

Comment:
nit: can you wrap `len(full_title) > TRACKING_MAX_FORUM_TITLE` in parens to make it clear that we're using at the boolean result of the `>` check?